forms/NumericField.php

```diff
@@ -4,138 +4,196 @@
  * Text input field with validation for numeric values. Supports validating
  * the numeric value as to the {@link i18n::get_locale()} value, or an
  * overridden locale specific to this field.
- * 
+ *
  * @package forms
  * @subpackage fields-formattedinput
  */
 class NumericField extends TextField {
-
 	/**
-	 * Override locale for this field
-	 * 
+	 * Override locale for this field.
+	 *
 	 * @var string
 	 */
 	protected $locale = null;
 
+	/**
+	 * @param mixed $value
+	 * @param array $data
+	 *
+	 * @return $this
+	 *
+	 * @throws Zend_Locale_Exception
+	 */
 	public function setValue($value, $data = array()) {
 		require_once "Zend/Locale/Format.php";
 
-		// If passing in a non-string number, or a value
```

Comment:
Not sure why this comment was removed; This was here to help explain the logic in this method for future developers and I think it still makes sense to keep that justification in place.